homeassistant/components/tplink/light.py

```diff
@@ -269,6 +285,18 @@ def supported_features(self):
         """Flag supported features."""
         return self._light_features.supported_features
 
+    def _get_valid_temperature_range(self):
+        """Return the device-specific white temperature range (in Kelvin).
+
+        :return: White temperature range in Kelvin (minimum, maximum)
+        """
+        for obj, temp_range in TPLINK_KELVIN.items():
+            if re.match(obj, self.smartbulb.sys_info[LIGHT_SYSINFO_MODEL]):
+                return temp_range
+        # pyHS100 is abandoned, but some bulb definitions aren't present
+        # use "safe" values for something that advertises color temperature
+        return (FALLBACK_MIN_COLOR, FALLBACK_MAX_COLOR)
```

Comment:
```suggestion
        return FALLBACK_MIN_COLOR, FALLBACK_MAX_COLOR
```
There is no need to have parentheses for tuples in python.

Reply:
Also, could you also add a test case for this as indicated by codecov?

Reply:
Fixed